astropy/coordinates/sky_coordinate.py

```diff
@@ -90,6 +91,20 @@ def _represent_as_dict(self):
         return out
 
 
+def _normalize_representation_type(kwargs):
```

Comment:
Move this to baseframe, use it there as well, and import it here? Would help ensure that we deprecate at the same time.